### DISCUSSION TITLE
which is your favorit ORM for use with Hyperdiv

### DISCUSSION THREAD

**Discussion by axelmoe** [hyperdiv/hyperdiv#67] [General]
Hi,

which is your favorit ORM for use with Hyperdiv? 
e.g. Gino, SQLModel .......?
KR Axel

**Comment by aldrichwright** [comment]
I'm a relative newbie to python so forgive the question but:

Both Gino and SqlModel are built on top of SQLAlchemy, right? Do the higher-level abstractions 'hide' useful features of the underlying tool? SQLAlchemy seems simple enough, is the additional ease of use of SQLModel say, enough to learn a new tool (I assume you'll need to know something about SQLALchemy if something goes wrong or SQLModel hides something that you need to get to).
